packages/core/bootstrap/src/lib/ws/index.ts

```diff
@@ -11,18 +11,20 @@ export * as reducer from './reducer'
 export * as epics from './epics'
 
 // TODO: WebSockets
-export const withWebSockets = (store: Store<RootState>): Middleware => async (execute) => async (
+export const withWebSockets = (store: Store<RootState>) => (wsHandler?: WSSubscriptionHandler): Middleware => async (execute) => async (
```

Comment:
Any reason for currying here? Seems like we could just use a 2 parameter function instead

Reply:
No specific reason yet, but I feel is good to have this flexibility. Makes the middleware composable and more testable.

Reply:
Sure, probably wont matter too much as soon as we get DI setup anyway